sdk/docs/manually-written/overview/overview/unifi.rst

```diff
@@ -92,26 +69,30 @@ Unifi
       * - `AWS KMS, GCP KMS <https://docs.daml.com/canton/usermanual/kms/kms.html>`__ and  `KMS Driver <https://docs.daml.com/canton/usermanual/kms/kms_driver_guide.html>`__
         - Yes
         - No
-      * - Postgres Backend
+      * - Packaged up private BFT synchronizer for private sub-networks
         - Yes
+        - No (Global Synchronizer Distribution only)
+      * - High-performance, highly available Database synchronizer for private sub-networks
         - Yes
+        - No
+      * - 24x7x365 SLA-based support 
+        - Yes
+        - No
   
-  Please follow below instructions in order to obtain your copy of Canton.
   
   .. _downloading:
   
-  Downloading Unifi
-  *****************
+  Obtain the enterprise distribution
+  **********************************
   
   Enterprise releases are available on request (sales@digitalasset.com) and can be downloaded from the
   respective `repository <https://digitalasset.jfrog.io/artifactory/canton-enterprise/>`__, or you can use
   our Canton Enterprise Docker images as described in our :brokenref:`Docker instructions <docker-instructions>`.
```

Comment:
let's comment out the brokenref part and tag as "TODO", as it won't work for now. It's pretty bad sales having this broken link.